src/sentry/consumers/__init__.py

```diff
@@ -23,6 +23,7 @@
     validate_consumer_definition,
 )
 from sentry.consumers.validate_schema import ValidateSchema
+from sentry.ingest.types import ConsumerType as IngestConsumerType
```

Comment:
Why alias?  `ConsumerType` seems like a fine name for this use-case.

Reply:
Noticed this type is defined in ingest. So this is in case we make ConsumerType definitions in other locations, in the future. I don't mind much either way though, that's just my reasoning

Reply:
hm i guess people adding a new type in the future can make the alias